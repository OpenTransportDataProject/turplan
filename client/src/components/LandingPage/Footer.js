import React from "react";
import styled from "styled-components";
import ntnu from "./images/ntnu_logo.png";
import sintef from "./images/sintef_logo.png";
import vegvesen from "./images/vegvesen_logo.png";

const FooterContainer = styled.div`
  flex: 1;
  display: flex;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
  background-color: rgb(58, 77, 109);
  padding-left: 10px;
  padding-right: 10px;
`;

const FooterText = styled.p`
  font-size: 1em;
  color: black;
  padding: 10px;
`;

const CustomerContainer = styled.div`justify-content: center;`;

const NTNUImg = styled.img`
  height: 50px;
  padding-right: 100px;
`;

const SINTEFImg = styled.img`
  height: 50px;
  padding: 10px;
`;

const VegvesenImg = styled.img`
  height: 50px;
  padding: 10px;
`;

export function Footer() {
  return (
    <FooterContainer>
      <NTNUImg src={ntnu} alt="ntnu" />
      <FooterText>
        {
          "TDT4290 Kundestyrt Prosjekt i samarbeid med SINTEF og Statens Vegvesen"
        }
      </FooterText>
      <CustomerContainer>
        <SINTEFImg src={sintef} alt="sintef" />
        <VegvesenImg src={vegvesen} alt="vegvesen" />{" "}
      </CustomerContainer>
    </FooterContainer>
  );
}

export default Footer;
