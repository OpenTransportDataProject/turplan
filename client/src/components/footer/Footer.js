import React from "react";
import ntnu from "./images/ntnu_logo.png";
import sintef from "./images/sintef_logo.png";
import vegvesen from "./images/vegvesen_logo.png";

import { FooterContainer, FooterText, CustomerContainer, NTNUImg, SINTEFImg, VegvesenImg } from './styles';

export function Footer() {
  return (
    <FooterContainer>
      <NTNUImg src={ntnu} alt="ntnu" />
      <FooterText>
        {
          "TDT4290 Kundestyrt Prosjekt i samarbeid med SINTEF Digital og Statens Vegvesen"
        }
      </FooterText>
      <CustomerContainer>
        <SINTEFImg src={sintef} alt="sintef" />
        <VegvesenImg src={vegvesen} alt="vegvesen" />{" "}
      </CustomerContainer>
    </FooterContainer>
  );
}

export default Footer;
