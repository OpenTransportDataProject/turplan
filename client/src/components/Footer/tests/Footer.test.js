import React from 'react';
import Footer from '../Footer.js';
import renderer from 'react-test-renderer';

test('Renders correctly', () => {
  const component = renderer.create(
    <Footer />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
