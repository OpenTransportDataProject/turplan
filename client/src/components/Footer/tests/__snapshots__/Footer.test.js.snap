// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div
  className="sc-bdVaJa dDPzOW"
>
  <img
    alt="ntnu"
    className="sc-bxivhb giPjmN"
    src="/images/ntnu_logo.png"
  />
  <p
    className="sc-bwzfXH fuhOkh"
  >
    TDT4290 Kundestyrt Prosjekt i samarbeid med SINTEF Digital og Statens Vegvesen
  </p>
  <div
    className="sc-htpNat jUffPd"
  >
    <img
      alt="sintef"
      className="sc-ifAKCX cSEGXs"
      src="/images/sintef_logo.png"
    />
    <img
      alt="vegvesen"
      className="sc-EHOje brCLHT"
      src="/images/vegvesen_logo.png"
    />
     
  </div>
</div>
`;
